README.md

```diff
@@ -177,7 +177,7 @@ A selection of websites offering high quality video stock.
 completely free.
 * [Videezy](https://www.videezy.com) - [:copyright:](https://www.videezy.com/terms) Free stock videos, 4k footage, and free HD video clips.
 * [Vidsplay](http://www.vidsplay.com/) - [:copyright:](http://www.vidsplay.com/terms.html) Free HD stock footage (Attribution required).
-* [VYOO](http://www.veed.io/vyoo) - Free Vertical Videos by [VEED.IO](http://www.veed.io/).
+* [VYOO](http://www.veed.io/vyoo) - Free Vertical Videos.
```

Comment:
@sabbakeynejad Still no link to licenses. If one is not provided I will close the PR.

Reply:
Hey I have submitted my PR for this, should be all good. Can we close this one pls?

Reply:
Fantastic, thanks!